includes/rest-api/class-sensei-rest-api-v1.php

```diff
@@ -61,12 +80,20 @@ public function get_endpoints() {
 		);
 	}
 
+	/**
+	 * @deprecated 3.11.0
+	 */
 	public function can_use_rest_api() {
 		$rest_api_enabled = Sensei()->feature_flags->is_enabled( 'rest_api_v1' );
```

Comment:
Should we also add the `_deprecated_function` here?

Reply:
We shouldn't as they would create warnings in the logs.